packages/connect/src/api/getAccountInfo.ts

```diff
@@ -67,7 +67,7 @@ export default class GetAccountInfo extends AbstractMethod<'getAccountInfo', Req
             // validate path if exists
             let address_n: number[] = [];
             if (batch.path) {
-                address_n = validatePath(batch.path, 3);
+                address_n = validatePath(batch.path);
```

Comment:
it might make sense, but I would still prefer not to do any changes beyond affecting other coins in this PR. 

could you please change it to something like 
```
                address_n = validatePath(batch.path, coinInfo.name !== 'solana'? 3: undefined);
```
( I did not test)